rules/config

```diff
@@ -174,10 +174,25 @@ KUBE_DOCKER_PROXY = http://172.16.1.1:3128/
 # These are Used *only* when INCLUDE_KUBERNETES=y
 # NOTE: As a worker node it has to run version compatible to kubernetes master.
 #
-KUBERNETES_VERSION = 1.21.1
-KUBERNETES_CNI_VERSION = 0.8.7
+KUBERNETES_VERSION = 1.21.1-00
+KUBERNETES_CNI_VERSION = 0.8.7-00
 K8s_GCR_IO_PAUSE_VERSION = 3.4.1
```

Comment:
Seem you do not need to move this block.

Reply:
Xuhui has a comment, he said the version number is not correct, adding the postfix should be good, he asked me help change this code.

Reply:
Changed